src/main/java/news/blog/com/controller/ArticleController.java

```diff
@@ -23,7 +26,7 @@ public String getArticles(Model model)
     @GetMapping("/{id}")
     public String getArticle(@PathVariable Long id, Model model)
     {
-        model.addAttribute("article", articleService.getArticle(id));
+        model.addAttribute("article", articleService.getArticleDto(id));
```

Comment:
let's develop our backend just as REST service w/o any views or frontend
model usage is unnecessary